src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVService/CVImageResult.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text;
+using RegressionGames.StateRecorder.JsonConverters;
+using UnityEngine;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models.CVSerice
```

Comment:
Same here, won't leave the comment in other places

Reply:
resolved